### DISCUSSION TITLE
Can not create a new "ceph block pool" from  cr?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10724] [Q&A]
if the status of the ceph cluster is warnning, can not create a new block pool?
https://github.com/rook/rook/blob/768602bbe8a1c060db4b29363d1b4ace9c38201f/pkg/operator/ceph/pool/controller.go#L181

**Comment by Madhu-1** [comment]
The ceph cluster is expected to be in a HEALTH_OK/HEATH_WARN state for creating new resources,

**Reply by Madhu-1** [reply]
https://github.com/rook/rook/blob/55a8fb9b8e1e89767ee70a0e16ae2fbc12c71ba2/pkg/operator/ceph/controller/controller_utils.go#L151

**Comment by zhucan** [comment]
![image](https://user-images.githubusercontent.com/13997135/184866396-a4369b6b-0ca0-4dfd-b2a4-0f8557b85aa6.png)
![image](https://user-images.githubusercontent.com/13997135/184866469-e6720f2a-7028-4993-85d6-7d6c8faf4465.png)
 
The status of ceph cluster is "HEALTH_WARN", the pool status is empty.
